fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1420,784 @@ def draw_act(self, obj, layout, value):
+            tmp_obj_name = ""
+        split_1.label(text=f"or {repr(tmp_obj_name)} if no name")
+
+    def draw_bhv_options(self, layout):
+        if self.export_all_selected:
+            return
+        box = layout.box()
+        prop_split(box, self, "preset_behavior_script", "Preset Behavior Script")
+        box.operator("bone.add_behavior_script", text="Add Behavior Cmd").option = len(self.behavior_script)
+        for index, bhv in enumerate(self.behavior_script):
+            bhv.draw(box, index)
+
+    def draw(self, layout):
+        # level exports
+        col = layout.column()
+        box = col.box()
+        box.operator("object.sm64_export_level", text="Export Level")
+        prop_split(box, self, "level_name", "Level")
+        if self.level_name == "custom":
+            prop_split(box, self, "custom_export_name", "Level Name")
+        self.draw_level_path(box.box())
+        col.separator()
+        # object exports
+        box = col.box()
+        if not self.export_col and not self.export_bhv and not self.export_gfx:
+            box2 = col.box()
+            box2.label(text="You must enable at least one export type", icon="ERROR")
+            self.draw_generic_options(box)
+            return
+        else:
+            box.operator("object.sm64_export_combined_object", text="Export Object")
+            self.draw_generic_options(box)
+
+        # bhv export only, so enable bhv draw only
+        if not self.export_col and not self.export_gfx:
+            return self.draw_bhv_options(col)
+
+        # pathing for gfx/col exports
+        prop_split(box, self, "export_header_type", "Export Type")
+
+        if self.export_header_type == "Custom":
+            prop_split(box, self, "custom_export_path", "Custom Path")
+
+        elif self.export_header_type == "Actor":
+            prop_split(box, self, "group_name", "Group Name")
+            if self.group_name == "custom":
+                prop_split(box, self, "custom_export_name", "Group Name")
+        else:
+            box.label(text="Destination level selection is shared with level export dropdown", icon="PINNED")
+            prop_split(box, self, "level_name", "Level")
+        if self.export_col and not self.export_all_selected:
+            prop_split(box, self, "collision_object", "Collision Obj")
+        if self.export_gfx and not self.export_all_selected:
+            prop_split(box, self, "graphics_object", "Geo Layout Obj")
+        if self.export_gfx and self.export_script_loads:
+            prop_split(box, self, "model_id", "Model ID Num")
+
+        # behavior options
+        if self.export_bhv and not self.export_all_selected:
+            self.draw_bhv_options(col)
+
+        # info/warnings
+        if self.export_header_type == "Custom":
+            info_box = box.box()
+            info_box.label(text="Export will not write any headers or dependencies", icon="ERROR")
+
+        if self.export_all_selected:
+            info_box = box.box()
```

Comment:
Should you not use .column() here?

Reply:
box is a valid sublayout that defines vertical orientation. There is no need to add in an extra `.column()`

> box()[](https://docs.blender.org/api/current/bpy.types.UILayout.html#bpy.types.UILayout.box)
> Sublayout (items placed in this sublayout are placed under each other in a column and are surrounded by a box)
> 
> Returns: Sub-layout to put items in
> 
> Return type: [UILayout](https://docs.blender.org/api/current/bpy.types.UILayout.html#bpy.types.UILayout)